src/gettrail_print.c

```diff
@@ -0,0 +1,146 @@
+#define _XOPEN_SOURCE 700
+#define  _BSD_SOURCE
+
+#include <stdio.h>
+#include <traildb.h>
+#include <Judy.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include "safeio.h"
+
+typedef uint64_t timestamp_t;
+
+void print_trails(char **traildb_paths, int num_paths, Pvoid_t cookies) {
+    printf("[\n");
```

Comment:
I think we can save some significant bytes if we don't use `\n` to format the output.

Reply:
Oh yeah, I mainly added this for debugging.